src/util-napatech.c

```diff
@@ -59,7 +59,7 @@ int NapatechIsBypassSupported(void)
  *
  * \return count of the Napatech adapters present in the system.
  */
-static int GetNumAdapters(void)
+int NapatechGetNumAdapters(void)
```

Comment:
This is called several times but the value is invariant. Suggest fetching value once and returning that value on subsequent calls.